starcheck/utils.py

```diff
@@ -245,9 +248,20 @@ def _yagzag_to_pixels(yag, zag):
     return row.tolist(), col.tolist()
 
 
-def _guide_count(mags, t_ccd, count_9th=False):
+def _t_ccds_bonus(mags, t_ccd, date):
+    dyn_bgd_dt_ccd = -4.0
+    if CxoTime(date).date < '2023:139':
+        dyn_bgd_n_faint = 0
+    else:
+        dyn_bgd_n_faint = 2
     eff_t_ccd = get_effective_t_ccd(t_ccd)
-    return float(guide_count(np.array(mags), eff_t_ccd, count_9th))
+    t_ccds_bonus = get_t_ccds_bonus(mags, eff_t_ccd, dyn_bgd_n_faint, dyn_bgd_dt_ccd)
+    return t_ccds_bonus
+
+
+def _guide_count(mags, t_ccd, date, count_9th=False):
```

Comment:
It's never a good idea to change the order of positional arguments. I guess this is a private function but still.